tests/vexflow_test_helpers.ts

```diff
@@ -185,6 +185,7 @@ export class VexFlowTests {
     Bravura: ['Bravura', 'Gonville', 'Custom'],
     Gonville: ['Gonville', 'Bravura', 'Custom'],
     Petaluma: ['Petaluma', 'Gonville', 'Bravura', 'Custom'],
+    Leland: ['Leland', 'Bravura', 'Gonville', 'Custom'],
```

Comment:
We probably don't need 'Gonville' here. Since Bravura is a complete SMuFL font, removing Gonville shouldn't have any impact on the tests.

Reply:
I also thought so, but I made the test and `vexWiggleArpeggioUp` is only defined in `Gonville`.

:( No change done.

Reply:
@0xfe I have seen that `vexWiggleArpeggioUp` is a 90 degrees rotated version of `wiggleArpeggiatoUp` does Vexflow provide runtime rotation or do theses glyphs have to be rotated offline?

Reply:
They have to be rotated. See https://github.com/w3c/smufl/issues/218#issuecomment-1146567666